plugins/source/hubspot/resources/services/crm/cursor.go

```diff
@@ -0,0 +1,27 @@
+package crm
+
+import (
+	"context"
+	"fmt"
+	"github.com/cloudquery/cloudquery/plugins/source/hubspot/client"
+)
+
+func setCursor(ctx context.Context, cqClient *client.Client, key string, cursor string) error {
+	if cqClient.Backend != nil && len(cursor) > 0 {
+		if err := cqClient.Backend.SetKey(ctx, key, cursor); err != nil {
```

Comment:
we should also prepend the key with the prefix (`c.ID()`?), if the user has different hubspot syncs).
See aws implementation for this

Reply:
Done in [602cc6c](https://github.com/cloudquery/cloudquery/pull/17470/commits/602cc6c03fc88a6d35901a8b83d811dc1bf77e1f)